### DISCUSSION TITLE
What to check to solve 'err_connection_refused' error

### DISCUSSION THREAD

**Discussion by unhyif** [vercel/community#6820] [Help]
### Summary

Sometimes I and some users experience the `ERR_CONNECTION_REFUSED` error.
I checked the Logs page on Vercel, but I couldn't see any error codes. Every request was responded to with a 200 status code.
I couldn't check the console for details because I was on mobile when I experienced the error.
I can't grasp what I need to check.

I'm using the Vercel hobby plan and not using custom domain. I didn't perform any network setups.
Usage overview seems okay.

<img width="930" alt="image" src="https://github.com/vercel/community/assets/93528293/5d938dd3-63f3-40d1-a6bc-74e0a0563043">

When I use https://downforeveryoneorjustme.com/ to check the server status, it seems okay too. (but I coudn't check it when the error occurred)

### Steps to Reproduce

Can't reproduce 😕

**Comment by amyegan** [comment]
Hey @unhyif. This error can happen for a variety of reasons, such as local network issues, a misconfigured domain, proxy issues, invalid credentials for an external service, or many other possible causes.

The problem seems to be intermittent so the most likely cause is a network connection issue. If you're able to try from a different network, that can be used to check if the problem is coming from your network.

Our [domain troubleshooting](https://vercel.com/docs/projects/domains/troubleshooting#) tips help with common domain issues. If you haven't already seen that page, it's worth a try.

I'd be happy to take a closer look if you need another set of eyes on it. Just share a link to your site so I can try to see the error
